### PR TITLE
Remove Tilt warning

### PR DESCRIPTION
**Pull request by Sajiyah-Salat**
Fixes: #3219
This pr just remove the tilt warning about port mapping. I have done what I did understand. If any futher changes is required to take kindly guide me and I am happy to that. Thank you. 
/kind cleanup

### CODE REVIEW

**Comment by mboersma** [comment]
/lgtm

I tested this locally with `make tilt-up` and it does make the warning disappear. 👍🏻

**Comment by mboersma** [comment]
/hold

This may have changed the local behavior; I'm seeing a 404 now for prometheus...

**Comment by mboersma** [comment]
/lgtm cancel

We were led astray by the warning message that referenced port 8080. Prometheus is actually listening at container port 9090:

```
[prometheus] ts=2023-03-07T19:36:54.749Z caller=web.go:561 level=info component=web msg="Start listening for connections" address=0.0.0.0:9090
[prometheus] ts=2023-03-07T19:36:54.749Z caller=main.go:993 level=info msg="Starting TSDB ..."
[prometheus] ts=2023-03-07T19:36:54.750Z caller=tls_config.go:232 level=info component=web msg="Listening on" address=[::]:9090
[prometheus] ts=2023-03-07T19:36:54.750Z caller=tls_config.go:235 level=info component=web msg="TLS is disabled." http2=false address=[::]:9090
```

If I change it to be explicit about both ports being 9090, things are happy:

```python
port_forwards = [port_forward(local_port = 9090, container_port = 9090, name = "View metrics")],
```

File: Tiltfile [hunk 1]
```diff
@@ -166,7 +166,7 @@ def observability():
     k8s_resource(
         workload = "prometheus-operator",
         new_name = "metrics: prometheus-operator",
-        port_forwards = [port_forward(9090, name = "View metrics")],
+        port_forwards = [port_forward(9090, 8080, name = "View metrics")],
```
Comment (mboersma) [inline Tiltfile:169]:
```suggestion
        port_forwards = [port_forward(local_port = 9090, container_port = 9090, name = "View metrics")],
```

**Comment by Sajiyah-Salat** [comment]
let me know if further changes is needed. and why my test are failing?

**Review by mboersma — COMMENTED** [review]
/lgtm
/retest

Thanks @Sajiyah-Salat! The aks e2e test is failing because it's not 100% reliable (we're working on it), not because of this PR.

**Comment by CecileRobertMichon** [comment]
/lgtm
/approve

Thank you and welcome @Sajiyah-Salat!

**Comment by CecileRobertMichon** [comment]
/retest

e2e should be fixed now

**Pull request merged**